File: Javascript/Fibonacci Sequence.js
function checking() {
    if(document.getElementById("num-input").value<=0)
    {
	    document.getElementById("mydiv").textContent= "the number smaller than zero";
	    
    }
    else
    {
        var num =[0,1];
    var first =0;
    var last =1;
    var i=0;
    while (i<document.getElementById("num-input").value-2)
    {
        last +=first;
        num.push(last);
        first = (first-last) * -1;
        i+=1;
        console.log(num)
    }
    var mydiv = document.getElementById("mydiv").textContent=num;
    }
    
}

File: Python/Fibonacci Sequence.py
count = int(input("enter the number you want to show\n"))
if count<=0:
    print("the number smaller than zero")
else:
    num =[0,1]
    first =0
    last =1
    i=0
    while i<count-2:
        last +=first
        num.append(last)
        first = (first-last) * -1
        i+=1

    for number in num:
        print(number)